config/rbac/imagejob_pods_cluster_role.yaml

```diff
@@ -3,31 +3,3 @@ kind: ClusterRole
 metadata:
   creationTimestamp: null
   name: imagejob-pods-cluster-role
-rules:
```

Comment:
looks like this is empty now?

Reply:
I think we need to keep it for the narrow-scoped service account for the imagejob pods to keep it separate from eraser-controller-manager. It doesn't need any permissions now that imagcollector is removed